tests/resources/parser/test_data/ca/TheGlobeAndMail.json

```diff
@@ -0,0 +1,37 @@
+{
+  "V1": {
+    "authors": [
+      "Reuters"
+    ],
+    "body": {
+      "summary": [],
+      "sections": [
+        {
+          "headline": [],
+          "paragraphs": [
+            "Cannabis firm Canopy Growth WEED-T said on Friday its CEO, David Klein, will retire at the end of the company’s current fiscal year on March 31.",
+            "Klein was appointed as Canopy Growth’s chairman when the then co-CEO and founder Bruce Linton was fired in 2019.",
+            "The announcement comes just days after the company reported a smaller-than-expected core loss and pointed to easing doubts around its ability to continue as a going concern.",
+            "The company had raised doubts about its ability to continue as a going concern last year as it struggled to turn profitable. It later sold its sports nutrition products unit, Biosteel, after seeking bankruptcy protection for it.",
+            "Canopy Growth said it is in the process of securing a search firm to initiate a comprehensive CEO selection process."
+          ]
```

Comment:
Would be nice to use an article including all selectable content (subheadline, paragraphs) as test case.

Reply:
I'll keep that in mind for future publishers as well :)